src/committee/orchestrator.rs

```diff
@@ -36,12 +40,235 @@ use super::node::{Round2Request, Round2Response};
 // Orchestration logic
 //
 
+type RpcOrchestratorContext = Arc<(Orchestrator, Arc<RwLock<MemberStatusState>>)>;
```

Comment:
Tried to do that but had some issues with jsonrpsee's context automatically wrapping the struct passed with an Arc. I tried to avoid double Arcs but it wasn't really possible since I already needed an Arc of the RwLock before passing it to the `RpcModule`, and the `RpcModule` has not function to fetch a clone of the Arc. But hey, we have double Arc anyway so we can definitely do what you suggested.